### ISSUE TITLE
Delete "six" as a dependency.

### ISSUE THREAD

**Issue by DailyDreaming** [HumanCellAtlas/dcp-cli#435]
We no longer support python 2.7, so this should be removed.  Depends on #430.

**Comment by jessebrennan** [comment]
Why do this as a separate issue?

**Comment by DailyDreaming** [comment]
@jessebrennan I made a comment in the PR (#430).
